resources/lib/favorites.py

```diff
@@ -24,8 +24,7 @@ def __init__(self, _kodi):
 
     def is_activated(self):
         ''' Is favorites activated in the menu and do we have credentials ? '''
-        from tokenresolver import TokenResolver
-        return self._kodi.get_setting('usefavorites') == 'true' and self._kodi.credentials_filled_in() and TokenResolver(self._kodi).logged_in()
+        return self._kodi.get_setting('usefavorites') == 'true' and self._kodi.credentials_filled_in()
```

Comment:
Great, I didn't want to mess with that as I did not fully understand why this was there.

Reply:
It was there to prevent the follow/unfollow function from failing if the user filled in wrong credentials and cancelled login.
But, this is a very rare situation and if it happens the user also gets a notification.
There is room to improve user experience in such rare cases, but this is not a priority.